.github/workflows/prod_deploy.yml

```diff
@@ -0,0 +1,75 @@
+name: Prod Deployment
+
+on:
+  workflow_run:
+    workflows: ["Dev Deployment"]
+    types:
+      - completed
+env:
+  IMAGE_TAG: ${{ github.sha }}
+
+jobs:
+  deploy:
+    name: Deploy infrastructure and chart
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    runs-on: runs-on
+    permissions:
+      id-token: write
+      contents: read
+    environment:
+      name: prod
+    steps:
+      - name: Sleep for 1 minute
+        run: echo "Waiting for 60 seconds to allow for abort of prod deployment" && sleep 60
+        shell: bash
+
+      - uses: actions/checkout@v4
+
+      - uses: citizensadvice/python-poetry-setup-action@v1
+        with:
+          install_cdk: true
+          project_root: "./cdk"
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: "arn:aws:iam::${{ vars.AWS_ACCOUNT }}:role/LocalOfficeSearchApiDeployment"
+          role-session-name: local-office-search-api-deploy-prod-workflow
+          aws-region: eu-west-1
+
+      - name: Deploy CDK
+        run: |
+          set -eo pipefail
+          cd ./cdk
+          poetry run cdk deploy "prod/**" LocalOfficeSearchApiEcrRepo --concurrency 2 --ci --require-approval never
```

Comment:
Is it correct to include `LocalOfficeSearchApiEcrRepo` in here? I always get confused by the ECR repo because it gets deployed to the dev account, but as part of the production stack.

Reply:
ah no it's not - I actually took this out of the stack definition because the repo already exists and isn't managed by CDK so trying to make it CDK managed was a bit too much so it shouldn't be in here at al